dev-python/nbdime/nbdime-3.1.1-r2.ebuild

```diff
@@ -0,0 +1,108 @@
+		-> ${P}-jupyter_server2-2.patch
+	https://github.com/jupyter/nbdime/commit/c4e1ddd0125a4358b19c980ebfe896a081446b8a.patch
+		-> ${P}-jupyter_server2-3.patch
+	https://github.com/jupyter/nbdime/commit/2f4bf944058048c18202edccb9f43ac4ec1de4fb.patch
+		-> ${P}-jupyter_server2-4.patch
+	https://github.com/jupyter/nbdime/commit/369bce46f2784a345900e42a656d5a9d05ef7e41.patch
+		-> ${P}-jupyter_server2-5.patch
+	https://github.com/jupyter/nbdime/commit/73b70c3bf32cf3dbac9b6660ad61dfd9fc7c64eb.patch
+		-> ${P}-jupyter_server2-6.patch
+	https://github.com/jupyter/nbdime/commit/5c5fda9547eb84c1a38acd2446b99e301b8cda29.patch
+		-> ${P}-jupyter_server2-7.patch
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="webtools"
+
+RDEPEND="
+	dev-python/colorama[${PYTHON_USEDEP}]
+	dev-python/GitPython[${PYTHON_USEDEP}]
+	dev-python/jinja[${PYTHON_USEDEP}]
+	dev-python/jupyter_server[${PYTHON_USEDEP}]
+	dev-python/jupyter_server_mathjax[${PYTHON_USEDEP}]
+	dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/tornado[${PYTHON_USEDEP}]
+	webtools? ( net-libs/nodejs[npm] )"
+BDEPEND="
+	test? (
+		dev-python/jsonschema[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/notebook[${PYTHON_USEDEP}]
+		dev-python/pytest-timeout[${PYTHON_USEDEP}]
+		dev-python/tabulate[${PYTHON_USEDEP}]
+	)"
+
+distutils_enable_sphinx docs/source \
+		dev-python/recommonmark \
+		dev-python/sphinx-rtd-theme
+distutils_enable_tests pytest
+
+PATCHES=(
+	"${DISTDIR}"/${P}-jupyter_server2-1.patch
+	"${DISTDIR}"/${P}-jupyter_server2-2.patch
+	"${DISTDIR}"/${P}-jupyter_server2-3.patch
+	"${DISTDIR}"/${P}-jupyter_server2-4.patch
+	"${DISTDIR}"/${P}-jupyter_server2-5.patch
+	"${DISTDIR}"/${P}-jupyter_server2-6.patch
+	"${DISTDIR}"/${P}-jupyter_server2-7.patch
+)
+
+EPYTEST_DESELECT=(
+	nbdime/tests/test_decision_tools.py::test_build_diffs_unsorted
+	nbdime/tests/test_diff_json_conversion.py::test_diff_to_json_patch
+	nbdime/tests/test_merge_notebooks.py::test_merge_conflicts_get_diff_indices_shifted
+	nbdime/tests/test_merge_notebooks.py::test_merge_inserts_within_deleted_range
+	nbdime/tests/test_merge_notebooks.py::test_merge_interleave_cell_add_remove
+	nbdime/tests/test_merge_notebooks.py::test_merge_multiline_cell_source_conflict
+	nbdime/tests/test_merge.py::test_deep_merge_lists_delete_no_conflict__currently_expected_failures
+	nbdime/tests/test_server_extension.py::test_diff_api_checkpoint
+	nbdime/tests/test_web.py::test_api_diff
+	nbdime/tests/test_web.py::test_api_merge
+	nbdime/tests/test_web.py::test_fetch_diff
+	nbdime/tests/test_web.py::test_fetch_merge
+	nbdime/tests/test_web.py::test_offline_mathjax
+)
+
+src_prepare(){
+	# convert CRLF to LF
+	sed -i \
+		-e 's/\r$//' \
+		nbdime/tests/conftest.py \
+		nbdime/tests/test_cli_apps.py \
+		nbdime/tests/test_web.py \
+		nbdime/webapp/nbdimeserver.py \
+		|| die 'sed failed'
```

Comment:
I suppose you can use edos2unix.eclass for this.

Reply:
Thank you for letting me know about this eclass !